src/azure-cli-core/azure/cli/core/config_defaults.py

```diff
@@ -0,0 +1,264 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+config = {
+    "core": {
+        "output": {
+            "default": "json",
+            "allowed": [
+                {
+                    "value": "json",
+                    "description": "JSON formatted output that most closely matches API responses."
+                },
+                {
+                    "value": "jsonc",
+                    "description": "Colored JSON formatted output that most closely matches API responses."
+                },
+                {
+                    "value": "table",
+                    "description": "Human-readable output format."
+                },
+                {
+                    "value": "tsv",
+                    "description": "Tab- and Newline-delimited. Great for GREP, AWK, etc."
+                },
+                {
+                    "value": "yaml",
+                    "description": "YAML formatted output. An alternative to JSON. Great for configuration files."
+                },
+                {
+                    "value": "yamlc",
+                    "description": "Colored YAML formatted output. An alternative to JSON. Great for configuration files."
+                },
+                {
+                    "value": "none",
+                    "description": "No output, except for errors and warnings."
+                }
+            ],
+            "type": "string",
+            "description": "The default output format."
+        },
+        "disable_confirm_prompt": {
+            "default": "false",
+            "allowed": [
+                {
+                    "value": "true",
+                    "description": "Commands will not wait for user confirmation."
+                },
+                {
+                    "value": "false",
+                    "description": "Commands will wait for user confirmation."
+                }
+            ],
+            "type": "boolean",
+            "description": "Turn confirmation prompts on/off"
+        },
+        "collect_telemetry": {
+            "default": "false",
```

Comment:
I think the default is `true` for `collect_telemetry`.

Reply:
fixed in e21cdea